hack/prom-rule-ci/prom-rules-tests.yaml

```diff
@@ -639,3 +639,49 @@ tests:
             value: 103706112
           - labels: 'kubevirt_vmi_memory_used_bytes{container="virt-handler", name="vm-example-2", namespace="default", node="node-1"}'
             value: 234329980
+
+  # Excessive VMI Migrations in a period of time
+  # Simulating metric resets at time slots 5 and 9
+  - interval: 1h
+    input_series:
+      - series: 'kubevirt_migrate_vmi_succeeded_total{vmi="vmi-example-1", source="node-1", target="node-2"}'
+        # time:  0 1 2 3 4 5 6 7 8 9 10 11 12 13
+        values: "_ 1 2 2 3 _ 1 2 2 _ _  1  2  3+0x13"        # 7 increases, in samples: 2,4,6,7,11,12,13
```

Comment:
@orenc1  Its 8 increases in total no?

Reply:
That is and important detail and this is why the unit testing can become biased.

Reply:
no, because the first one doesn't count (change from `_` to `1`).
but it does count on the second series, since there is already a datapoint for `kubevirt_migrate_vmi_succeeded_total{vmi="vmi-example-1"}`